docs/doc_5_best_practices.md

```diff
@@ -28,6 +28,45 @@ if test != ""
     requestBody.StringPointerValue = test
 ```
 
+## Working with Arrays and Slices
+
+When working with arrays in Golang we need to consider various scenarios such as the absence of a struct property  or property being null.
```

Comment:
Mostly because of semantics of go and Atlas API.

The problem can be seen in this playground example:
https://go.dev/play/p/a0y4DOHEHtX

That is the situation after this PR fix. 

https://go.dev/play/p/mRMMVjdNLwr

I have provided some context in PR description and thread offline.